docs/cimpl6ToolingReference.md

```diff
@@ -58,7 +58,7 @@ SHR-CLI produces one or all of the following outputs, depending on configuration
 * [FHIR profiles, extensions, value sets](#fhir-export) that form the core content of the IG,
 * A [Logical Model](#logical-model-export) corresponding to the CIMPL class definitions, expressed as FHIR **StructureDefinition**,
 * [JSON Schema](#json-schema-export) for the profiles defined by the IG,
-* A [Data Dictionary](#data-dictionary-export) that lists the [**MustSupport**](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) elements in the IG, as well as value sets and value set members,
+* A [Data Dictionary](#data-dictionary-export) that lists profiles defined as part of the IG, profile elements marked as [**MustSupport**](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) in the IG [Content Profile](#content-profile-file), and their value sets and value set members,
```

Comment:
This revision strongly clarifies what to expect as the contents of the data dictionary, which was a previous point of confusion for me. Thank you!

Reply:
Thanks Mint!